### PR TITLE
Add aarch64

### PR DESCRIPTION
**Pull request by Tobias-Fischer**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
@conda-forge-admin, please rerender

**Comment by Tobias-Fischer** [comment]
@conda-forge/torchvision this is ready :). Thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -118,6 +118,7 @@ outputs:
         # 2022/03/29 hmaarrfk
         # It seems that this test can cause segmentation faults on the CIs.
         {% set tests_to_skip = tests_to_skip + " or test_write_video_with_audio" %}
+        {% set tests_to_skip = tests_to_skip + " or test_video_clips_custom_fps" %}
```
Comment (h-vetinari) [inline recipe/meta.yaml:121]: Let's not skip aarch-specific tests globally please (also below)
```suggestion
        {% set tests_to_skip = tests_to_skip + " or test_video_clips_custom_fps" %}  # [aarch64]`
```

**Comment by h-vetinari** [comment]
Do we want to include the CUDA builds in this PR already?

**Comment by h-vetinari** [comment]
Otherwise this looks pleasantly painless so far, thanks for giving this a shot @Tobias-Fischer!

File: recipe/meta.yaml (hunk 1 above)
Comment (Tobias-Fischer) [reply recipe/meta.yaml:121]: This test actually failed on Linux-64 .. not on aarch

**Comment by Tobias-Fischer** [comment]
Let’s do cuda builds separately

File: recipe/meta.yaml (hunk 1 above)
Comment (Tobias-Fischer) [reply recipe/meta.yaml:121]: See https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=644049&view=logs&jobId=9a864fd9-6c8f-52ca-79ce-2aa6dca1a1de

**Comment by Tobias-Fischer** [comment]
Thankas for the quick review @h-vetinari! Please take a look at my comments

File: recipe/meta.yaml (hunk 1 above)
Comment (h-vetinari) [reply recipe/meta.yaml:121]: Interesting. Why didn't they fail before then? I see no changed host dependency based on the pinnings here.

Comment (Tobias-Fischer) [reply recipe/meta.yaml:121]: I guess it’s just a sporadic failure

Comment (Tobias-Fischer) [reply recipe/meta.yaml]: Launched another run without the skip, let's see what happens.

**Comment by Tobias-Fischer** [comment]
Alright, so the `test_video_clips_custom_fps` test now succeeded in the linux-64 configuration where it had previously failed (see https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=644144&view=logs&j=9a864fd9-6c8f-52ca-79ce-2aa6dca1a1de&t=10fc5aa2-324e-5982-4c88-6b31fcab16b3) but failed on linux-aarch64 where it previously passed (compare https://github.com/conda-forge/torchvision-feedstock/runs/10744660735 with https://app.travis-ci.com/github/conda-forge/torchvision-feedstock/jobs/593688462). Therefore, I think we should disable this test - it's sporadically failing on different platforms, so probably in previous builds we just got lucky ..

File: recipe/meta.yaml (hunk 1 above)
Comment (Tobias-Fischer) [reply recipe/meta.yaml:121]: See https://github.com/conda-forge/torchvision-feedstock/pull/64#issuecomment-1396753883 - resolving here.

Comment (hmaarrfk) [reply recipe/meta.yaml:121]: I think their av integration is buggy.

Comment (Tobias-Fischer) [reply recipe/meta.yaml:121]: Agreed. Are we happy to merge then @hmaarrfk @h-vetinari?

**Review by h-vetinari — APPROVED** [review]

**Pull request merged**